### ISSUE TITLE
"Attach Node Script" popup is stretched across entire viewport height

### ISSUE THREAD

**Issue by Xrayez** [godotengine/godot#48481]
**Godot version:**
3.4.beta 1209ee0df

**OS/device including version:**
Windows 10.

**Issue description:**

![image](https://user-images.githubusercontent.com/17108460/117166270-e2827200-adce-11eb-8a4a-ec68c857e2a5.png)

**Steps to reproduce:**
1. Create new scene with a root node.
2. Try to attach a new script to root node.

**Minimal reproduction project:**
Not needed.

**Comment by akien-mga** [comment]
Possibly a regression from my cherry-pick of #47686? (d304187357594283ca419cf349cf92fd64f483ce)
CC @aaronfranke

**Comment by kleonc** [comment]
Changing:
https://github.com/godotengine/godot/blob/d304187357594283ca419cf349cf92fd64f483ce/editor/script_create_dialog.cpp#L788
to:
```
script_name_warning_label->set_autowrap(false);
```
results in:
![FX200lLTc7](https://user-images.githubusercontent.com/9283098/117181692-cf26d500-add5-11eb-9d21-f97f7d94073a.png)

**Comment by akien-mga** [comment]
Fixed by #48494.

**Comment by kleonc** [comment]
It's happening again in 6355877c0d6410f45d84cd13ed1ec4e462efeb9f (didn't check which exact commit caused it).

**Comment by Calinou** [comment]
@kleonc Can you try to [bisect the regression](https://docs.godotengine.org/en/latest/community/contributing/bisecting_regressions.html) to determine when this started happening again? This would be greatly appreciated :slightly_smiling_face:

**Comment by kleonc** [comment]
Ok, I wanted to bisect it now but got kinda strange result right at the start - for me it seems to not be fixed even in e82f56e3a901e5988226ac1c067fbc90430c0eba which was merged as a fix for this issue:

![PpORmEhPfW](https://user-images.githubusercontent.com/9283098/118722574-97377d00-b82c-11eb-9c32-118d3e2edc2b.png)
![godot windows tools 64_vFRoa2TNX8](https://user-images.githubusercontent.com/9283098/118722551-9272c900-b82c-11eb-807d-0c7dfa101bb9.png)

So I have no idea what's going on. I think I've compiled `3.x` many times in the last 13 days (that's when that fix was committed) and I don't think I've seen this bug until today (but I'm not 100% sure). Could it be something on my end? :thinking: Could someone else test this?

**Comment by Xrayez** [comment]
I confirm it still happens on `3.x` branch.

**Comment by kleonc** [comment]
For convienience I've also checked that for me in 917630107ca0196d330a1a620b543f400ad40ba9 it works properly and in d304187357594283ca419cf349cf92fd64f483ce bug is present (everything as expected). So probably nothing wrong on my end and e82f56e3a901e5988226ac1c067fbc90430c0eba simply didn't fix that bug at all.

**Comment by aaronfranke** [comment]
This bug is really weird. It seemed to work when I submitted #48494, but it does indeed seem broken now.. I decided to run this test code:

```cpp
	print_line(get_size());
	print_line(alert->get_size());
	print_line(builtin_warning_label->get_size());
	print_line(script_name_warning_label->get_size());
```

And what I got was this:

```
374, 252
341, 110
9, 1937
9, 1853
```

It seems that for some reason as soon as the script create dialog is created, most of its elements have 0 size. This causes the labels to auto-wrap immediately, so that the text would be like this:

```
W
a
r
n
i
n
g
```

I don't really understand why this is happening. I don't understand why the labels are given a chance to do grow so tall before their parents are sized, or why the size of the parent won't shrink down afterwards, or why this bug doesn't happen with the nearly-identical `builtin_warning_label` but does happen with the new `script_name_warning_label`, or why `get_size()` reports a sane size that doesn't include the label's size and is inconsistent with the observed size. Also, this bug doesn't exist in 4.0, might be because the labels are handled differently there somehow.

Anyway, I tried setting up anchors/margins/etc so that the parents are grown correctly etc, but it didn't work. the only fix I could come up with was forcing the labels to be wide at the beginning, to avoid them from growing tall in the first place: #48824

**Comment by aaronfranke** [comment]
Judging by the fact that nobody has complained since #48824 was merged, I'll close this and assume that #48824 fixed it.

Although, I am still not sure what the root cause of this issue was and why it only occurs on 3.x, so if anyone wants to investigate further they are welcome to.
